src/components/headerImageCaption.tsx

```diff
@@ -62,11 +63,12 @@ const HeaderImageCaptionStyles = css`
 `;
 
 const svgStyle = css`
-	line-height: 3.4375rem;
+	line-height: 32px;
 	font-size: 0;
 	svg {
-		width: 1.35rem;
-		height: 1.35rem;
+		width: 24px;
+		height: 24px;
+		margin: 4px;
```

Comment:
I assume use of px is more consistent cross browser/platform?
Interesting, will this affect other parts  of the code using rem/em?